incubator/cerebro/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  - name: common
```

Comment:
I'd suggest not to use the common chart. It's future is kind of unclear, plus, it is in incubator and will stay there. Usually, I suggest to move charts directly to stable, but stable charts are not allowed to depend on incubator charts.

Reply:
oh no - I liked the style of less typing - maybe some notes should be put into it to avoid it's usage?

Reply:
fixed